src/utils/jwt.ts

```
@@ -1,44 +1,121 @@
 /**
  * JWT helpers for 2FA challenge tokens.
- * Reuses JWT_SECRET from config; challenge tokens are short-lived (e.g. 5m).
+ * Challenge tokens use dedicated config and include aud/iss claims for purpose binding.
+ * These tokens CANNOT be used for API access and have strict expiration (5m).
+ * 
```

ESLint

[error] Delete `·`

(prettier/prettier)